unidist/core/backends/mpi/core/controller/garbage_collector.py

```diff
@@ -63,14 +63,17 @@ def _send_cleanup_request(self, cleanup_list):
         # Cache serialized list of data IDs
         s_cleanup_list = SimpleDataSerializer().serialize_pickle(cleanup_list)
         async_operations = AsyncOperations.get_instance()
+        mpi_state = communication.MPIState.get_instance()
+        cur_rank = mpi_state.rank if mpi_state is not None else 0
```

Comment:
Why do we need to check if mpi_state is not None? Isn't it always not None?

Reply:
I copied it from a logger initializing, but I agree that is unnecessarily here.